packages/truffle-core/cli.js

```diff
@@ -11,6 +11,7 @@ if (nodeMajorVersion < 8) {
   process.exit(1);
 }
 
+const Config = require("truffle-config");
```

Comment:
Unused global require var. 
```suggestion
```

Looks like you merged `next` after already committing this change so `eslint` didn't catch it.